beacon-chain/core/validators/committees_test.go

```diff
@@ -54,7 +54,7 @@ func TestExceedingMaxValidatorsFails(t *testing.T) {
 
 func BenchmarkMaxValidators(b *testing.B) {
 	var validators []*pb.ValidatorRecord
-	validator := &pb.ValidatorRecord{WithdrawalShard: 0}
+	validator := &pb.ValidatorRecord{Pubkey: []byte{'A'}}
```

Comment:
Does this need to have any data anyway? Can it be `&pb.ValidatorRecord{}` throughout these tests?